pkg/config/config.go

```diff
@@ -52,6 +53,17 @@ type BackupS3 struct {
 	Region  string `yaml:"region"`
 }
 
+type BackupScp struct {
+	Host       string `yaml:"host"`
+	Port       string `yaml:"port"`
+	User       string `yaml:"user"`
+	Path       string `yaml:"path"`
+	SshKey     string `yaml:"sshKey"`
+	PassPhrase string `yaml:"passPhrase"`
```

Comment:
PasaPhrase and Password should not be raw because someone can access to imprtant data (ex: production db) using `backup` attributes.
It prefer to use env.